Assignment3.txt

```diff
@@ -0,0 +1,8 @@
+Anya Lauria
+
+1. The code is invalid. This is because while the print statement is trying to output a variable which has not been 
```

Comment:
Correct! The assumption that would also be needed for this code to run is the initialized i that's greater than 0.